Dockerfile

```diff
@@ -18,11 +18,17 @@ RUN cargo chef cook --release --recipe-path recipe.json
 COPY . .
 RUN cargo build --release --bin recert
 
+FROM golang:1.19-bookworm as ouger-builder
+COPY . .
+RUN git submodule update --init --recursive
```

Comment:
I think it's best to put this responsibility on whoever is running the Dockerfile